sfge/src/main/java/com/salesforce/config/SfgeConfig.java

```diff
@@ -16,4 +16,16 @@ public interface SfgeConfig {
 
     /** Indicates if a Jorje parse error causes the entire process to stop. */
     boolean shouldIgnoreParseErrors();
+
+    /**
+     * Indicates if Warn level logs to log4j should be forwarded to CLI as well when verbose is
+     * enabled
+     */
+    boolean shouldLogWarningsOnVerbose();
+
+    /**
+     * Should be used to set the level of increments at which path analysis progress update is
+     * provided
+     */
+    int getProgressIncrementsOnVerbose();
```

Comment:
Unnecessary, since we're making the progress report happen all the time instead of just on Verbose.

Reply:
Since the functionality has been modified, this can be renamed to not include "OnVerbose". Though I still think this is a useful variable to have so that users could increase or decrease depending on their convenience. For now, I haven't surfaced these new variables as `scanner:run:dfa`'s flags.